charts/scalardb-cluster/README.md

```diff
@@ -40,10 +39,10 @@ Current chart version is `1.0.0-SNAPSHOT`
 | scalardbCluster.image.repository | string | `"ghcr.io/scalar-labs/scalardb-cluster-node"` | Docker image reposiory of ScalarDB Cluster. |
 | scalardbCluster.image.tag | string | `""` | Override the image tag whose default is the chart appVersion |
 | scalardbCluster.imagePullSecrets | list | `[]` | Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace. |
+| scalardbCluster.logLevel | string | `"INFO"` | The log level of ScalarDB Cluster |
```

Comment:
The `scalardbCluster.storageConfiguration.*` configurations are deprecated now. And before, we use `scalardbCluster.storageConfiguration.dbLogLevel` to set the log level (configuration for `log4j2.properties).

However, I think the log level configuration is not a `Storage` configuration. So, I renamed it as `scalardbCluster.logLevel`.